### PR TITLE
operator-sdk 0.19.0

### PR DESCRIPTION
**Pull request by SeekingMeaning**
Created with `brew bump-formula-pr`.

### CODE REVIEW

**Comment by chenrui333** [comment]
```
Testing operator-sdk
/usr/bin/sandbox-exec -f /private/tmp/homebrew20200710-90418-bkgisn.sb ruby -W0 -I $LOAD_PATH -- /usr/local/Homebrew/Library/Homebrew/test.rb /usr/local/Homebrew/Library/Taps/homebrew/homebrew-core/Formula/operator-sdk.rb --verbose
==> /usr/local/Cellar/operator-sdk/0.19.0/bin/operator-sdk version
==> /usr/local/Cellar/operator-sdk/0.19.0/bin/operator-sdk new test --repo=github.com/example-inc/app-operator
Error: unknown flag: --repo
Usage:
  operator-sdk new <project-name> [flags]
```

File: Formula/operator-sdk.rb [hunk 1]
```diff
@@ -53,13 +53,9 @@ def install
...
-
-    cd "test" do
-      # Add an example API resource. This exercises most of the various pieces
-      # of generation logic.
-      system "#{bin}/operator-sdk", "add", "api", "--api-version=app.example.com/v1alpha1", "--kind=AppService"
-    end
+    # Create an example AppService operator. This exercises most of the various pieces
+    # of generation logic.
+    args = ["--type=ansible", "--api-version=app.example.com/v1alpha1", "--kind=AppService"]
+    system "#{bin}/operator-sdk", "new", "test", *args
```
Comment (chenrui333) [inline Formula/operator-sdk.rb:59]: anything that we can assert after the system call?

Comment (SeekingMeaning) [reply Formula/operator-sdk.rb:59]: I think it creates some files

Comment (SeekingMeaning) [reply Formula/operator-sdk.rb:59]: Added a check for requirements.yml

These files/directories are also created:

- build/
- deploy/
- molecule/
- roles/
- watches.yaml

Comment (SMillerDev) [reply Formula/operator-sdk.rb:59]: One of them is good enough. I doubt it'll be a packaging issue if it stops creating just one of these.

**Review by claui — APPROVED** [review]

**Comment by claui** [comment]
Thanks @SeekingMeaning!

**Pull request closed without merging**